app/forms/claim_form.rb

```diff
@@ -0,0 +1,45 @@
+class ClaimForm
+  include ActiveModel::Model
+  STEPS = %w[providers mentors].freeze
+
+  attr_accessor :claim, :step, :claim_params
+
+  def save
+    claim.assign_attributes(claim_params)
+    return false unless valid?
+
+    claim.save!
+  end
+
+  def valid?
+    claim.valid? && valid_providers? && valid_mentors?
+  end
```

Comment:
👍 I think form objects should mirror active record methods as you've done. Nice!